packages/haul-core-legacy/package.json

```diff
@@ -26,11 +26,10 @@
   "dependencies": {
     "mime-types": "^2.1.24",
     "mkdirp": "^0.5.1",
-    "ws": "^2.2.2",
+    "ws": "^6.2.1",
```

Comment:
good to know this is not breaking anything

Reply:
I test it manually and it's fine.